docs/howtoguides/enable_fips.md

```diff
@@ -24,6 +24,12 @@ this operation makes changes to underlying SSL-related libraries and requires a
 reboot into the FIPS-certified kernel.
 ```
 
+```{important}
+Once you enable FIPS, enabling some Pro services will not be possible.
```

Comment:
For both `fips` and `realtime-kernel`, I avoided mentioning that the services can be disabled, as we still leave the kernel around after disable which still conflicts with some services. But let me know what you think

Reply:
That makes sense. If they can't be cleanly disabled, it's probably safer to not bring the ability to disable them to users' attention or we might cause them problems. Since these services are aimed at enterprise users, if they want to disable it they can presumably contact support to be walked through that process?

Reply:
Yes, there is ways to disable those services, it is just that it is not that straightforward as you mentioned.